include/net/nrf_cloud_rest.h

```diff
@@ -86,18 +91,14 @@ struct nrf_cloud_rest_context {
 	size_t response_len;
 };
 
-/** @brief Data required for nRF Cloud location request */
-struct nrf_cloud_rest_location_request {
-	/** Unique device identifier; null-terminated */
-	char *device_id;
-	/** Network information used in location request */
+/** @brief Data required for nRF Cloud cellular positioning request */
+struct nrf_cloud_rest_cell_pos_request {
+	/** Network information used in request */
 	struct lte_lc_cells_info *net_info;
 };
 
 /** @brief Data required for nRF Cloud Assisted GPS (A-GPS) request */
 struct nrf_cloud_rest_agps_request {
-	/** Unique device identifier; null-terminated */
-	char *device_id;
```

Comment:
How is the device id indicated now?  Implicitly through the JWT?

Reply:
correct